finance_manager/src/main/java/com/exadel/finance/manager/wallet/controller/WalletController.java

```diff
@@ -0,0 +1,58 @@
+package com.exadel.finance.manager.wallet.controller;
+
+import com.exadel.finance.manager.mapstruct.mappers.MapStructMapper;
+import com.exadel.finance.manager.wallet.model.entity.Wallet;
+import com.exadel.finance.manager.wallet.service.api.WalletService;
+import com.sandbox.api.WalletsApi;
+import com.sandbox.model.WalletRequestDto;
+import com.sandbox.model.WalletResponseDto;
+import java.util.List;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/")
```

Comment:
Is that final version of api path?

Reply:
Other mappings can be seen in the interface